apps/store/src/app/[locale]/layout.tsx

```diff
@@ -54,6 +81,32 @@ export const generateStaticParams = () => {
 
 export default Layout
 
+export const metadata = {
```

Comment:
We could use the `Metadata` type from `next` to validate right?
```suggestion
export const metadata: Metadata = {
```